### PR TITLE
chm-415 capacity building resource view page

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrate  capacity building resource view page

### Designs
same as abach project design

### Testing instructions
image

### CODE REVIEW

File: app/views/forms/view/chm/capacity-building-resource.vue [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                </ul>
+            </div>
+
+            <div v-if="document.bchSubjects">
+                <legend>{{t("cartagenaProtocol")}}</legend>        
+                <div v-if="document.bchSubjects">
+                    <label>{{t("thematicAreass")}}</label>
+                    <ul class="km-value">
+                        <li v-for="term in document.bchSubjects">
+                            <km-term :value="term" :locale="locale"></km-term>
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/capacity-building-resource.vue:180]: We also have a very useful directive, 'view-terms-hierarchy', if you want to show the terms hierarchy

 <view-terms-hierarchy ng-model="document.cbdSubjects" locale="locale" term-domain="cbdSubjects"
       on-terms-loaded="onCbdSubjectsTerms(terms)"></view-terms-hierarchy>

Comment (liyuanmontreal) [reply app/views/forms/view/chm/capacity-building-resource.vue:180]: fixed,many thanks!

**Comment by liyuanmontreal** [comment]
image
image
image

File: app/views/forms/view/chm/capacity-building-resource.vue [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+                    <div class="km-value">                           
+                        <ng v-vue-ng:view-terms-hierarchy  v-model:ng-model="document.cbdSubjects" locale="locale"   term-domain="cbdSubjects"></ng>
+                    </div>
+                </div>
+        
+                <div v-if="document.gbfTargets">
+                    <div v-if="document.gbfTargets">
+                        <label>{{t("gbfTarget")}}</label>                        
+                        <div class="km-value">                           
+                            <ng v-vue-ng:view-terms-hierarchy  v-model:ng-model="document.gbfTargets" locale="locale"  term-domain="gbfTargets"></ng>
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/capacity-building-resource.vue:147]: Is "gbfTargets" a term?

Comment (liyuanmontreal) [reply app/views/forms/view/chm/capacity-building-resource.vue]: fixed, change to not use view-terms-hierarchy

**Pull request closed without merging**